contributors/design-proposals/job.md

```diff
@@ -79,8 +109,21 @@ type JobSpec struct {
     // job should be run with. Defaults to 1.
     Completions *int
 
+    // Optional duration in seconds relative to the startTime that the job may be active
+    // before the system tries to terminate it; value must be a positive integer.
+    ActiveDeadlineSeconds *int
+
+    // Optional number of retries before marking this job failed.
+    BackoffLimit *int
+
+    // Optional time (in seconds) specifying how long a job should be retried before marking it failed.
+    BackoffDeadlineSeconds *int
```

Comment:
ProgressDeadlineSeconds to match the field in Deployments? Seems like it's the same concept although in Deployment we merely add a condition in the DeploymentStatus.